libs/pxweb2-ui/src/lib/components/VariableBox/VariableBox.tsx

```diff
@@ -52,6 +52,9 @@ export function VariableBox({
   const chosenValuesLength = selectedValues.find(
     (variables) => variables.id === id
   )?.values.length;
+  const currentVariable = selectedValues.find((variable) => variable.id === id);
+  const hasMandatoryNoSelectedValuesError =
```

Comment:
Consider using another variable name. This one is a bit difficult to follow. Maybe isMissingMandatoryValueError?

Reply:
Changed to use this name